.github/scripts/utils.zsh/setup_linux

```diff
@@ -97,7 +97,7 @@ sudo apt-get install -y --no-install-recommends \
   libasound2-dev libfdk-aac-dev libfontconfig-dev libfreetype6-dev libjack-jackd2-dev \
   libpulse-dev libsndio-dev libspeexdsp-dev libudev-dev libv4l-dev libva-dev libvlc-dev \
   libpci-dev libdrm-dev \
-  nlohmann-json3-dev libwebsocketpp-dev libasio-dev libvpl-dev libqrcodegencpp-dev
+  nlohmann-json3-dev libwebsocketpp-dev libasio-dev libvpl-dev libvpl2 libqrcodegencpp-dev
```

Comment:
Why do we need to install `libvpl2` in addition to the dev package? Is that necessary for compilation?

Reply:
We need the library to link it. Since we only linked libdl previously and dont set errors on undefined functions it was fine to build with broken links.

For whatever reason ubuntu does not have a dependency between the -dev and actual library packages. So we need to manually specify it.